css/solar-system.css

```diff
@@ -6,6 +6,11 @@
   --gray4: #454e56;
   --gray5: #2a2e35;
   --gray6: #12181b;
+  --purple-extra-dark: #160336;
```

Comment:
I'm actually happy with these color names.

Reply:
I took into consideration when you mentioned about the number at the end of the colors. This time, because there are less shades I used words to name them. It fit quite nice in my opinion. However, I do understand what @ArchitectFromMars is trying to say there. Maybe we could use the suggested format to name the gray colors/many shades colors.